libvast/src/address.cpp

```diff
@@ -16,13 +16,103 @@
 
 #include <arpa/inet.h>
 #include <netinet/in.h>
+#include <openssl/crypto.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include <openssl/ossl_typ.h>
 #include <sys/socket.h>
 
 #include <cstdlib>
 #include <utility>
 
 namespace vast {
 
+namespace {
+inline uint32_t bitmask32(size_t bottom_bits) {
+  return bottom_bits >= 32 ? 0xffffffff : ((uint32_t{1} << bottom_bits) - 1);
+}
+
+class AddressEncryptor {
```

Comment:
Please use snake_case in the code base for consistency.

Reply:
Removed bad habit in 31020cabe